pallets/nft/src/lib.rs

```diff
@@ -450,11 +486,43 @@ pub mod pallet {
 		) -> DispatchResultWithPostInfo {
 			let sender = ensure_signed(origin)?;
 
+			ensure!(quantity >= 1, Error::<T>::InvalidQuantity);
+
 			Self::do_mint_nfts(&sender, class_id, metadata, attributes, quantity)?;
 
 			Ok(().into())
 		}
 
+		/// Minting new stackable NFTs using provided class ID, metadata,
+		/// attributes, and amount
+		///
+		/// The dispatch origin for this call must be _Signed_.
+		/// - `class_id`: class ID of the collection the NFT will be part of
+		/// - `metadata`: NFT assets metadata as NFT metadata
+		/// - `attributes`: NFTs' attributes
+		/// - `amount`: the balance of the minted stackable NFTs
+		///
+		/// Emits `NewStackableNftMinted` if successful.
+		#[pallet::weight(< T as Config >::WeightInfo::mint_stackable_nft())]
+		pub fn mint_stackable_nft(
+			origin: OriginFor<T>,
+			class_id: ClassIdOf<T>,
+			metadata: NftMetadata,
+			attributes: Attributes,
+			amount: BalanceOf<T>,
+		) -> DispatchResultWithPostInfo {
+			let sender = ensure_signed(origin)?;
+
+			ensure!(amount > Zero::zero(), Error::<T>::InvalidStackableNftAmount);
+
+			let result = Self::do_mint_nfts(&sender, class_id, metadata, attributes, 1)?;
+			StackableCollections::<T>::insert((class_id, result.1, sender.clone()), amount);
```

Comment:
Will value be replaced if key exits?

Reply:
Since minting stackable NFT mints a new token together with setting up  its balance replacing it should not occur but could add additional check